build_scripts/appimage/appimage.sh

```diff
@@ -61,75 +62,61 @@ setup_python_37_appdir() {
 }
 
 copy_server_config_to_build_dir() {
-    cp "$APPIMAGE_DIR"/server_config.json.standard "$1"/monkey_island/cc/server_config.json
+    cp "$APPIMAGE_DIR"/server_config.json.standard "$BUILD_DIR"/monkey_island/cc/server_config.json
 }
 
 install_monkey_island_python_dependencies() {
-  local appdir=$1
-  local build_dir=$2
   log_message "Installing island requirements"
 
   log_message "Installing pipenv"
-  "$appdir"/AppRun -m pip install pipenv || handle_error
+  "$APPDIR"/AppRun -m pip install pipenv || handle_error
 
-  requirements_island="$build_dir/monkey_island/requirements.txt"
-  generate_requirements_from_pipenv_lock "$appdir" "$build_dir" "$requirements_island"
+  requirements_island="$BUILD_DIR/monkey_island/requirements.txt"
+  generate_requirements_from_pipenv_lock "$requirements_island"
 
   log_message "Installing island python requirements"
-  "$appdir"/AppRun -m pip install -r "${requirements_island}"  --ignore-installed || handle_error
+  "$APPDIR"/AppRun -m pip install -r "${requirements_island}"  --ignore-installed || handle_error
 }
 
 generate_requirements_from_pipenv_lock () {
-  local appdir=$1
-  local build_dir=$2
-  local requirements_island=$3
+  local requirements_island=$1
 
   log_message "Generating a requirements.txt file with 'pipenv lock -r'"
-  pushd "$build_dir/monkey_island"
-  "$appdir"/AppRun -m pipenv --python "$appdir/AppRun" lock -r > "$requirements_island" || handle_error
+  pushd "$BUILD_DIR/monkey_island"
+  "$APPDIR"/AppRun -m pipenv --python "$APPDIR/AppRun" lock -r > "$requirements_island" || handle_error
   popd
 }
 
+
 install_mongodb() {
-  local build_dir=$1
-  local mongo_path="$build_dir/monkey_island/bin/mongodb"
   log_message "Installing MongoDB"
 
-  mkdir -p "$mongo_path"
-  "$build_dir/monkey_island/linux/install_mongo.sh" "${mongo_path}" || handle_error
-}
-
-add_monkey_icon() {
-  local appdir=$1
-  local monkey_repo=$2
-
-  unlink "$appdir"/python.png
-  mkdir -p "$appdir"/usr/share/icons
-  cp "$monkey_repo"/monkey/monkey_island/cc/ui/src/images/monkey-icon.svg "$appdir"/usr/share/icons/infection-monkey.svg
-  ln -s "$appdir"/usr/share/icons/infection-monkey.svg "$appdir"/infection-monkey.svg
+  mkdir -p "$MONGO_PATH"
+  "$BUILD_DIR/monkey_island/linux/install_mongo.sh" "${MONGO_PATH}" || handle_error
 }
 
-add_desktop_file() {
-  local appdir=$1
-
-  unlink "$appdir"/python*.desktop
-  cp ./infection-monkey.desktop "$appdir"/usr/share/applications
-  ln -s "$appdir"/usr/share/applications/infection-monkey.desktop "$appdir"/infection-monkey.desktop
-}
-
-add_apprun() {
-  cp ./AppRun "$1"
+remove_python_appdir_artifacts() {
+  unlink "$APPDIR"/python.png
+  unlink "$APPDIR"/python*.desktop
+  unlink "$APPDIR"/AppRun
```

Comment:
Why unlink instead or `rm`?

Reply:
Probably because some of them were symlinks, and I use `unlink` for symlinks instead of `rm`; not because it does anything substantively different, just because that's how my brain works. I cam make them `rm` instead.